src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,53 +1,110 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 /**
  * <p>Реалізувати свій ArrayList який імплементує інтерфейс List. Дотриматися основних вимог щодо
  * реалізації ArrayList (default capacity, newCapacity...)</p>
  */
 public class ArrayList<T> implements List<T> {
+    private static final int CAPACITY = 10;
+    private int size = 0;
+    private T [] arrayList;
+
+    public ArrayList() {
+        arrayList = (T[]) new Object[CAPACITY];
+    }
+
+    public ArrayList(int size) {
+        ensureCapacity(size);
+    }
+
+    public void ensureCapacity(int size) {
+        if (size >= CAPACITY) {
```

Comment:
CAPACITY will always stay 10, meaning you will do resize every add() method after your size is bigger than that
```suggestion
        if (size >= arryList.length) {
```

Reply:
yes, it makes no sense to compare with the original long constant